source/docs/yearly-overview/yearly-changelog.rst

```diff
@@ -109,6 +109,11 @@ General Library
 - ArcadeDrive: Fix max output handling
 - Add ``PWMSparkFlex`` Motor Controller
 - ADIS16470: allow accessing all three axes
+- Deprecated ``MotorControllerGroup``. Use ``PWMMotorController`` ``addFollower()`` method or if using CAN motor controllers use their method of following.
+- Added functional inteface to ``DifferentialDrive`` and ``MecanumDrive``. The ``MotorController`` interface may be removed in the future to reduce coupling with vendor libraries. Instead of passing ``MotorController`` objects, the following method references or lambda expressions can be used:
+
+    - Jave: ``DifferentialDrive drive = new DifferentialDrive(m_leftMotor::set, m_rightMotor::set);``
```

Comment:
```suggestion
    - Java: ``DifferentialDrive drive = new DifferentialDrive(m_leftMotor::set, m_rightMotor::set);``
```

Reply:
Or better yet we could use a code tab?

Reply:
I was worried about how code tabs would look in that list. Seems like it would break things up too much